README.md

```diff
@@ -0,0 +1,55 @@
+# testcafe-react-selectors
+
+An extension which simplifies the work with ReactJS components for [Tescafe](https://github.com/DevExpress/testcafe) e2e testing framework
+
+##Install
+
+`$ npm install testcafe-react-selectors`
+
+##Usage
+
+####Create selector for ReactJS component
+`ReactSelector` allows create testcafe `Selector` being passed name of component class or nested component element.
+E.g. `ReactSelector('TodoList')`, `ReactSelector('TodoList TodoItem')`, `ReactSelector('TodoItem span')`  etc.
+You are free combine this selectors with testcafe `Selector` filter function like `.withText`, `.nth` and [others](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#functional-style-selectors)
+
+```js
+import ReactSelector from 'testcafe-react-selectors';
+
+fixture('React application testing').page('http://localhost:1337');
+
+test('Add new item', async t => {
+    const getEl = ReactSelector('AddItemButton');
+
+    await t.click(getEl);
+
+    const iteamLabel = ReactSelector('Label div')
+
+    await t.expect(iteamLabel.textContent).eql('New Item');
+});
+```
+
+####Getting of component props and state
+
+As an alternative [testcafe snapshot properties](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html) you can get a `state` or `props` of ReactJS  component. It could be useful for checking assertion and in many cases simplifies an assertion logic.
+You can get these data from `react` property of `ReactSelector` snapshot.
+The following example illustrates how it can be used:
+
+```js
+import ReactSelector from 'testcafe-react-selectors';
+
+fixture('React application testing').page('http://localhost:1337');
+
+test('Add new item', async t => {
+    const statusBar = await ReactSelector('Status Bar').react;

```

Comment:
Yes, i agree. With current implementation we can't use:
```js
const statusBar = await ReactSelector('StatusBar');
const statusBarProps = await statusBar.react.props;
```
Because `props` is not shorthand. We define it in external library (this repo).
We can write:
```js
const statusBarReactProps = (await statusBar.react).props;
```
or
```js
const statusBarReactData =  await statusBar.react;
```
or
```js
const statusBarReact =  await statusBar.react;
```
What do you think @inikulin ?